domain/src/main/java/com/upf464/koonsdiary/domain/usecase/cotton/AddQuestionAnswerUseCase.kt

```diff
@@ -18,7 +19,7 @@ internal class AddQuestionAnswerUseCase @Inject constructor(
         }
 
         val questionAnswer = QuestionAnswer(
-            questionId = request.questionId,
+            questionId = Question(id = request.questionId, korean = "", english = ""),
```

Comment:
이렇게 하는 것보다 Question 클래스의 모든 변수에 초기값을 넣어주고 여기는 아래처럼 바꾸는게 나을 것 같아요.
```suggestion
            questionId = Question(id = request.questionId),
```